filters-share/specific_URL.txt

```diff
@@ -382,6 +382,7 @@
 /^https:\/\/thumb\.toomics\.com\/upload\/banner\/(?!main|cut)/
 ||ppss.kr/wp-content/uploads/2021/09/ktx_
 ||ppss.kr/wp-content/uploads/2022/02/cartax-
+||ppss.kr/wp-content/uploads/2022/10/pacey
```

Comment:
Please move to top of the files.

Reply:
And, update its version.

Reply:
All 3 filter or just the new one ?

Reply:
New one.

Reply:
i opened a new one because too many conflicts are here